flutter.el

```diff
@@ -103,8 +104,9 @@ the `flutter` process."
        (error "Root of Flutter project not found"))))
 
 (defmacro flutter--with-run-proc (args &rest body)
-  "Execute BODY while ensuring an inferior `flutter` process is running.
+  "Execute BODY while ensuring an inferior `flutter run` process is running.
 
+BUFFER-NAME is the chosen buffer where will run the proc.
```

Comment:
There is no `BUFFER-NAME` parameter, so I don't understand this.

Reply:
Sorry, I removed the code and forgot to undo this